src/main/java/no/digipost/print/validate/PdfValidator.java

```diff
@@ -288,8 +288,20 @@ private void validerSideantall(final int numberOfPages, int maxPages, final List
 	}
 
 	private boolean harTekstIStrekkodeomraade(final PDPage pdPage) throws IOException {
```

Comment:
Jeg skjønner at du beregner boksen for strekkode her, men det er veldig kompakt, så det blir litt vanskelig å forholde seg til alle målene. Kanskje det blir litt verbost, men går det an å dra noen av disse ut i variabler? Tenker på for eksempel `mmToPoints(pageHeightInMillimeters-BARCODE_AREA_WIDTH_MM)`. Eller gir disse veldig mening om man har intellisense i IDE?